src/lib/utils.ts

```diff
@@ -30,3 +30,6 @@ export type Payload = Encoded.Bytearray;
 
 const parseEnv = (x) => x && JSON.parse(x);
 export const presentInvalidTokens = parseEnv(process.env.SHOW_INVALID_TOKENS);
+
+export const contractIdToAccountId = (address: string) =>
```

Comment:
There are types for this as well. Check eg `Encoded.AccountAddress` from the SDK package. `Encoded` contains a few helpful types.